_data/retail.yml

```
@@ -161,6 +190,8 @@ websites:
     email_address: info@eventim.de
     img: eventim.png
     lang: de
+    regions:
+      - de
```

```suggestion
    lang: de
```
EVENTIM supports [22 countries](https://www.eventim.de/campaign/international/). Many regional sites appear to use the same login system.